examples/edhoc-rs-no_std/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "edhoc-rs-no_std"
+authors = ["Kaspar Schleiser <kaspar@schleiser.de>"]
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+edhoc-rs = { path = "../../lib", default-features = false, features = ["hacspec-cc2538"] }
+hexlit = "0.5.3"
+
+# depend on an allocator
+static-alloc = { version = "0.2.3" }
+
+# cortex-m specifics
+cortex-m = "0.7.4"
+cortex-m-rt = "0.7.1"
+cortex-m-semihosting = "0.5.0"
+panic-semihosting = { version = "0.6.0", features = ["exit"] }
+
+[patch.crates-io]
+hacspec-lib = { git = "https://github.com/malishav/hacspec", branch = "aesccm" }
```

Comment:
Is this necessary given that it is already present in the workspace `Cargo.toml`?

Reply:
It is if this example is excluded from the main workspace.
But I think I like better the solution in #25 there the crate is part of the workspace, but not part of the `default-members`.
I'll wait for #25 and see if that method works here, too!

Reply:
yes, that works. I've implemented it the same way (and removed this `[patch.crates-io]`).